extensions/machine-learning-services/src/controllers/mainController.ts

```diff
@@ -169,9 +165,9 @@ export default class MainController implements vscode.Disposable {
 	/**
 	 * Returns the server config manager instance
 	 */
-	public get serverConfigManager(): ServerConfigManager {
+	public get serverConfigManager(): PackageManagementService {
```

Comment:
why not get packagaeManagementService()?

Reply:
public get **serverConfigManager**(): ?

Reply:
I renamed the module and also renamed the property name here

---
In reply to: [395374192](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r395374192) [](ancestors = 395374192)